### ISSUE TITLE
Readme documentation - compilation error

### ISSUE THREAD

**Issue by gr9** [kmbulebu/dsc-it100-java#11]
Howdy,

Love the idea of this and am eagerly looking forward to using it (I've inherited a system in my new house & I'm wanting to test how the zones are mapped out/labelled).  I'm finding it a little hard to follow though; for starters - the top most code example in the ReadMe.md file doesn't compile; couple of different errors.  Obviously it's not a complete piece of code & it's lacking import statements etc.. but even once I've resolved those import dependancies with the obvious ones there's still a few things relating to 'Observable' and 'Action1' that are showing as compile errors for me.  Which could just be my lack of experience with Apache Mina and/or Java RX.

Inspecting the jar file of the latest release I see there's also an ExampleApp.class file; which must obviously compile otherwise it wouldn't be part of the jar; but my jar decompiler (JD-GUI) doesn't have any linkage for the two imports from that class: 

import rx.Observable;
import rx.functions.Action1;

Which means that the ExampleApp.class must be referring to two other dependancies that are required to be within the classpath but haven't been bundled into the jar.

Long story short- I can't figure out how to get this to work ;)

But in my defence, I think it's a documentation thing which is why I'm posting it here as an issue.

**Comment by gr9** [comment]
Nevermind.. decided in the case of dependancies I should probably just run it as a proper maven project rather than trying to tag the release jar blindly onto some code.. Go figure.. it seems to be working now ;)

**Comment by kmbulebu** [comment]
@gr9 Sorry for the slow response. I'm happy you got it working. 

@v3rm0n  Wrote an excellent example application, virtual-keypad:
https://github.com/v3rm0n/virtual-keypad
